loopy/target/__init__.py

```diff
@@ -159,6 +160,13 @@ def get_kernel_executor(self, kernel, *args, **kwargs):
         """
         raise NotImplementedError()
 
+    @abc.abstractproperty
+    def is_executable(self) -> bool:
```

Comment:
If `single_subkernel_is_entrypoint`, could you throw an error somewhere if more than one subkernel exists?

Reply:
Just to make sure that I understand this right: `single_subkernel_is_entrypoint == not is_executable `yes? @inducer

Reply:
> If single_subkernel_is_entrypoint, could you throw an error somewhere if more than one subkernel exists?

@sv2518 fixed this in #701.